src/scss/06_components/paragraphs/_accordion.scss

```diff
@@ -137,6 +137,10 @@
   > *:first-child {
     margin-top: 0;
   }
+
+  > .component--paragraph-text .component__container .component__content > .form-item:first-child > *:first-child {
```

Comment:
Oh boy this is quite hard to read 😂 Are you sure this is the only way to target what ever you are targeting? At least we need comment here that what does this do.

Reply:
Yeah it's quite a monster, isn't it..
I'll try to find better selector and if can't then I'll add a comment.
There was an unnecessary gap between the accordion heading and the content caused by the `<p>` tag's margin, but it should be only removed from the first one.

Reply:
@teroelonen I made the selector little less complicated and added a comment. If a better solution pops up in your mind, please tell! 🙂

Reply:
@dire this looks like something I can still manage - not easy but definitely better than before and the comment makes it easier to understand. Good work 👍